runtime/rastrace/method_trace.c

```diff
@@ -488,11 +488,18 @@ traceMethodArguments(J9VMThread* thr, J9UTF8* signature, UDATA* arg0EA, char* bu
 		switch (*sigChar) {
 		case '[':
 		case 'L':
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
use the -Xtrace option and see if you can triggered on a method with qtypes just to double check that this is all we need to get that working